sass/components/_asset-card.scss

```diff
@@ -14,7 +15,7 @@ $asset-card-padding: 0.4rem;
     grid-area: banner;
     display: block;
     position: relative;
-    aspect-ratio: 16/9;
+    aspect-ratio: 16 / 9;
```

Comment:
Is this meant to be a list or division?

Reply:
> **Nit**: `math.div` is not needed in this case, CSS accepts ratios in `x / y` form for `aspect-ratio`: https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio
> 
> I mean, you could use `math.div` as the property also accepts decimal numbers… but (for me at least) it just adds noise.

What Doup said above; so more like a list than division I think.

Reply:
Whoops, didn't see that above. Makes sense, though.

Reply:
If I understand correctly, is similar to saying `16:9`, but [CSS choose](https://drafts.csswg.org/css-values-4/#ratios) `/` to separate width/height (?).

You can also write it as `1.7777` which CSS would interpret as `1.7777 / 1` (?). The only thing… that's not as useful/familiar as saying `16:9`.